coqide.opam

```diff
@@ -23,7 +23,7 @@ depends: [
   "conf-adwaita-icon-theme"
   "coqide-server" {= version}
   "cairo2" {>= "0.6.4"}
-  "lablgtk3-sourceview3" {>= "3.1.2"}
+  "lablgtk3-sourceview3" {>= "3.1.5"}
```

Comment:
Maybe we should do something to be compatible with both lablgtk versions?

Reply:
I would rather think about back porting your patch as opam patch for all versions supported in Coq Platform. In the end it was my mistake years back to put the check into CoqIDE rather than into lablgtk and I think we should fix this mistake now and take none of it into the future.

Reply:
IMHO not worth supporting old versions Gaëtan

Reply:
changed so that the stricter constraint is only on windows

Reply:
@SkySkimmer : regarding old CoqIDE : the options are:

- tag a new patch release for older versions of CoqIDE
- do an opam level patch (keep the tag and add a patch file to opam) in the global opam repo
- as above, but do this in the Coq Platform opam patch repo

IMHO solution 2 is most appropriate. Solution 1 is probably overkill and solution 3 would exclude non platform users (should be rare on Windows, though).